external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/SingleTopicKafkaSpoutTest.java

```diff
@@ -1,379 +0,0 @@
-        List<KafkaSpoutMessageId> capturedMessageIds = messageIds.getAllValues();
-        spout.fail(capturedMessageIds.get(5));
-        spout.fail(capturedMessageIds.get(3));
-        spout.nextTuple();
-        spout.fail(capturedMessageIds.get(2));
-
-        //Check that the spout will reemit all 3 failed tuples and no other tuples
-        ArgumentCaptor<KafkaSpoutMessageId> reemittedMessageIds = ArgumentCaptor.forClass(KafkaSpoutMessageId.class);
-        for (int i = 0; i < messageCount; i++) {
-            spout.nextTuple();
-        }
-        verify(collector, times(3)).emit(anyString(), anyList(), reemittedMessageIds.capture());
-        Set<KafkaSpoutMessageId> expectedReemitIds = new HashSet<>();
-        expectedReemitIds.add(capturedMessageIds.get(5));
-        expectedReemitIds.add(capturedMessageIds.get(3));
-        expectedReemitIds.add(capturedMessageIds.get(2));
-        assertThat("Expected reemits to be the 3 failed tuples", new HashSet<>(reemittedMessageIds.getAllValues()), is(expectedReemitIds));
-    }
-
-    @Test
-    public void testShouldDropMessagesAfterMaxRetriesAreReached() throws Exception {
-        //Check that if one message fails repeatedly, the retry cap limits how many times the message can be reemitted
-        int messageCount = 1;
-        prepareSpout(messageCount);
-
-        //Emit and fail the same tuple until we've reached retry limit
-        for (int i = 0; i <= maxRetries; i++) {
-            ArgumentCaptor<KafkaSpoutMessageId> messageIdFailed = ArgumentCaptor.forClass(KafkaSpoutMessageId.class);
-            spout.nextTuple();
-            verify(collector).emit(anyString(), anyList(), messageIdFailed.capture());
-            KafkaSpoutMessageId msgId = messageIdFailed.getValue();
-            spout.fail(msgId);
-            assertThat("Expected message id number of failures to match the number of times the message has failed", msgId.numFails(), is(i + 1));
-            reset(collector);
-        }
-
-        //Verify that the tuple is not emitted again
-        spout.nextTuple();
-        verify(collector, never()).emit(any(), any(), any());
-    }
-
-    @Test
-    public void testSpoutMustRefreshPartitionsEvenIfNotPolling() throws Exception {
-        try (SimulatedTime time = new SimulatedTime()) {
-            SingleTopicKafkaUnitSetupHelper.initializeSpout(spout, conf, topologyContext, collector);
-
-            //Nothing is assigned yet, should emit nothing
-            spout.nextTuple();
-            verify(collector, never()).emit(anyString(), anyList(), any(KafkaSpoutMessageId.class));
-
-            SingleTopicKafkaUnitSetupHelper.populateTopicData(kafkaUnitRule.getKafkaUnit(), SingleTopicKafkaSpoutConfiguration.TOPIC, 1);
-            Time.advanceTime(KafkaSpoutConfig.DEFAULT_PARTITION_REFRESH_PERIOD_MS + KafkaSpout.TIMER_DELAY_MS);
-
-            //The new partition should be discovered and the message should be emitted
-            spout.nextTuple();
```

Comment:
Why delete the UT for single topic test?

Reply:
It was renamed here, but the old file wasn't deleted https://github.com/apache/storm/pull/2466/files#diff-b827d459a46f189dced7382d2d551439